src/main/java/academy/mate/bookshelf/controller/BookController.java

```diff
@@ -0,0 +1,49 @@
+package academy.mate.bookshelf.controller;
+
+import java.util.List;
+import javax.validation.Valid;
+import academy.mate.bookshelf.dto.BookDto;
+import academy.mate.bookshelf.entity.Book;
+import academy.mate.bookshelf.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shelf")
```

Comment:
@RomanMinevich Зазвичай коли ти будеш використовувати методи PUT, DELETE і передава и Id - то згідно принципам REST користувач (який буде викликати твій код) буде думати що він оновлює або відповідно видаляє полицю.... А це BookController і відповідно ви працюємо з книгами
```suggestion
@RequestMapping("/book")
```

Reply:
Замінив